governance.md

```diff
@@ -89,7 +89,7 @@ mechanics, such as mailing lists, meeting times, etc.
 
 More information:  
 [SIG Governance Requirements]  
-[SIG Lifecycle] - for a tactical checklist on creation and retirement  
+[SIG Lifecycle](https://github.com/kubernetes/community/blob/master/sig-wg-lifecycle.md) - for a tactical checklist on creation and retirement  
```

Comment:
> @nikhita are these generated files?

No :)

This change needs to be made at the bottom of this doc. There is a line which says `[SIG Lifecycle]: /sig-wg-lifecycle`. Looks like it just misses the `.md` extension :)

Reply:
Thank you @nikhita  :D